### PR TITLE
feat: populate jurisdictional preferences on settings page

### PR DESCRIPTION
**Pull request by emilyjablonski**
# Pull Request Template

## Issue Overview

This PR addresses #2751

- [x] This change addresses the issue in full

## Description

Fetches jurisdictional preferences from all the jurisdictions the user has on the settings page. I fought with this for a while trying to call the single hook multiple times and ended up just changing the comparison on the hook to `IN` so I could pass in multiple jurisdictions in one call. Open to feedback!

## How Can This Be Tested/Reviewed?

Log in as admin, ensure you can see all jurisdictions preferences. Create a parter and assign them one jurisdiction, ensure you can see only that jurisdictions preferences.

Ensure the loading state and empty states trigger appropriately.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] I have exported any new pieces added to ui-components
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

**Review by YazeedLoonat — CHANGES_REQUESTED** [review]
Hey @emilyjablonski this looks good and I think the `IN` instead of `=` in the hook works 

I do think we're missing a column on the table though, from the AC we also need a "jurisdiction" column which I didn't see

**Comment by emilyjablonski** [comment]
@YazeedLoonat Good call, that isn't in [the designs](https://www.figma.com/file/Y8bFd0aSz7eYJ2TCNid6s3/bloom-partners?node-id=5725%3A182450) so I'll check w Jesse!

File: ui-components/src/locales/general.json [hunk 1]
```diff
@@ -923,6 +923,7 @@
   "t.text": "Text",
   "t.time": "time",
   "t.to": "to",
+  "t.updated": "Updated",
```
Comment (emilyjablonski) [inline ui-components/src/locales/general.json:926]: Move to partners overrides

**Review by kramduckner — APPROVED** [review]
lgtm! One question about our git flows - re: the alerts from the onlick events - once this branch goes into dev, won't that hold up a release until those event handlers have something more functional?

**Comment by emilyjablonski** [comment]
@kramduckner good q! the link to the settings page only shows if `jurisdiction.enablePartnerSettings` is true, which we can turn off if this feature isn't complete for the next release

**Pull request merged**